pkg/network/payload/version.go

```diff
@@ -5,19 +5,13 @@ import (
 
 	"github.com/nspcc-dev/neo-go/pkg/config"
 	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/network/capability"
 )
 
-// Size of the payload not counting UserAgent encoding (which is at least 1 byte
-// for zero-length string).
-const minVersionSize = 27
-
-// List of Services offered by the node.
 const (
-	nodePeerService uint64 = 1
-	// BloomFilerService uint64 = 2 // Not implemented
-	// PrunedNode        uint64 = 3 // Not implemented
-	// LightNode         uint64 = 4 // Not implemented
-
+	// Size of the payload not counting UserAgent encoding (which is at least 1 byte
+	// for zero-length string).
+	minVersionSize = 27
```

Comment:
I don't see it being used in any way, do we still need it?

Reply:
No, we don't, removed this.